spec/models/service_order_spec.rb

```diff
@@ -18,6 +18,10 @@ def create_request
   end
   let(:tenant) { Tenant.seed }
 
+  it "accesses database when unchanged model is saved" do
+    expect { service_order.valid? }.to make_database_queries(:count => 26)
```

Comment:
will you reference service_order above this line - that way we are not testing the create, but rather the validation after the model has been saved

```suggestion
    service_order
    expect { service_order.valid? }.to make_database_queries(:count => 26)
```